src/components/UI/ThemeToggle/ThemeToggle.jsx

```diff
@@ -0,0 +1,9 @@
+export const ThemeToggle = () => {
+	return (
+		<>
+			<a className="header__links badge_base btn btn--primary btn--icon btn--circular theme__toggler">
```

Comment:
What's the use of using a tag here? You don't even have " href " attribute.

Reply:
This page needs to be worked on. I will update it when implementing the theme toggle feature.